src/calculateRentalCost.js

```diff
@@ -28,7 +28,17 @@
  */
 
 function calculateRentalCost(days) {
-  // write code here
+  let total = 0;
+
+  if (days < 3) {
+    total = 40 * days;
+  } else if (days >= 3 && days < 7) {
+    total = (40 * days) - 20;
+  } else {
+    total = (40 * days) - 50;
```

Comment:
20 and 50 are currently what's called a `magic numbers` - noone knows what they are
assign them to a variable to let your fellow-devs know what it is